_layouts/qa.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+color: no-logo program-page
+color: dark-nav
+---
+<div class='keyline-top pad8y qa-page'>
+  <div class='limiter'>
+    <div class='pad4y col12 clearfix prose min-height'>
+      <h1>{{page.title}}</h1>
+      {% if page.subtitle %}<h2 class="abstract-subtitle">{{page.subtitle}}</h2>{% endif %}
+
+      <section class='col3'>
+        <p>
+          <a href="/sessions/{{page.code}}" title="Page of the session with abstract and video">Back to session page</a>
+        </p>
+
+        <p>
+          <a href='/programme/#session_{{page.code}}'>Back to schedule</a>
```

Comment:
is this link really needed here? i mean you can get to the programme also from the nav bar, and since we're already 2 clicks away from the programme page here anyway there is not _so_ much need for a direct return link, I guess. but just my 5 cents

Reply:
Link removed. I just put it here since this was also on the session page - but yeah, it's not that useful, and I usually just use the browser's back button anyway.